<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignUp</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/gym2.jpg</normaloff>:/images/gym2.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Arial&quot;;

background-color: rgb(255, 255, 255);


color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.232955 rgba(255, 0, 0, 255), stop:0.801136 rgba(0, 0, 255, 255));</string>
    </property>
    <property name="title">
     <string>Register Here</string>
    </property>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Name :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ID">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>59</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(204, 204, 204);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>ID :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_phone">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>95</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Phone :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_email">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>131</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Email ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineName">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>39</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineID">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>75</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="linePhone">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>111</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEmail">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>147</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="SignUp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
font: 8pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;

color: rgb(0, 183, 134);
color: rgb(0, 170, 127);
</string>
    </property>
    <property name="text">
     <string>IIUC GYM CENTER</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
